py_swf/clients/workflow.py

```diff
@@ -65,3 +66,182 @@ def terminate_workflow(self, workflow_id, reason):
             workflowId=workflow_id,
             reason=reason,
         )
+
+    def count_open_workflow_executions(
+            self,
+            oldest_start_date,
+            latest_start_date=None,
+            workflow_name=None,
+            version=None,
+            tag=None,
+            workflow_id=None,
+    ):
+        """
+        Count the number of open workflows for a domain. You can pass in filtering criteria.
+        The results are best effort and may not exactly reflect recent updates and changes.
```

Comment:
Hmm you mean for the filter criteria or the results are best effort ? For filter criteria their descriptions are in the docstring. For best effort they kinda explain themselves? Also their is link to boto API just below this comment.